.github/CONTRIBUTING.md

```diff
@@ -56,6 +56,24 @@ approved by the reviewer.
 
 Once the pull request is approved, a team member will take care of merging.
 
+## Contributing training scripts
```

Comment:
Lets add a step:

- open an issue on GitHub to discuss the model and dataset you'd like to add a training script for.
- tag @ianjjohnson and @LukeWood in the issue desrciption